example.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Copyright (c) 2010, SoftLayer Technologies, Inc. All rights reserved.
```

Comment:
Ultra minor, since you updated the year on README.md and other places, might as well do so on this new file.